src/libNode/FinalBlockProcessing.cpp

```diff
@@ -824,8 +826,16 @@ bool Node::ProcessFinalBlock(const vector<unsigned char>& message,
 
     // Now only forwarded txn are left, so only call in lookup
     CommitMBnForwardedTransactionBuffer();
-    if (m_mediator.m_lookup->GetIsServer() && !isVacuousEpoch) {
-      m_mediator.m_lookup->SenderTxnBatchThread();
+    if (m_mediator.m_lookup->GetIsServer() && !isVacuousEpoch &&
+        !m_mediator.GetIsVacuousEpoch() &&
+        ((m_mediator.m_currentEpochNum + NUM_VACUOUS_EPOCHS + 1) %
```

Comment:
the lookup won't initiate the packet thread if it's an epoch earlier than the vacuous epoch